models/experimental/mamba/benchmarks/README.md

```diff
@@ -0,0 +1,23 @@
+# lm-eval-harness.py
+
+To run zero-shot evaluations of our model (to compare against against to Table
+3 of the Mamba paper), we use the lm-evaluation-harness library.
```

Comment:
can you add a reference to the Mamba paper as you are calling out a specific table from there.